packs/amp.js

```diff
@@ -3,17 +3,17 @@ const icon = `data:image/svg+xml,%3Csvg xmlns="http://www.w3.org/2000/svg" viewB
 
 const UIStrings = {
   /** Additional description of a Lighthouse audit that tells the user how they can improve image loading by using WebP in the context of AMP. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
-  'uses-webp-images': 'Consider displaying all your `amp-img` components in WebP formats while specifying an appropriate fallback for other browsers. [Learn more](https://amp.dev/documentation/components/amp-img/#example:-specifying-a-fallback-image).',
+  'uses-webp-images': 'Consider displaying all `amp-img` components in WebP formats while specifying an appropriate fallback for other browsers. [Learn more](https://amp.dev/documentation/components/amp-img/#example:-specifying-a-fallback-image).',
   /** Additional description of a Lighthouse audit that tells the user how images are automatically lazy loaded for the AMP framewok. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
-  'offscreen-images': 'Ensure that you are you using valid `amp-img` tags for your images which automatically lazy-load outside the first viewport. [Learn more](https://amp.dev/documentation/guides-and-tutorials/develop/media_iframes_3p/?format=websites#images).',
+  'offscreen-images': 'Ensure that you are using `amp-img` for images to automatically lazy-load. [Learn more](https://amp.dev/documentation/guides-and-tutorials/develop/media_iframes_3p/?format=websites#images).',
```

Comment:
```suggestion
  'offscreen-images': 'Ensure that you are using `amp-img` for images so that they automatically lazy-load. [Learn more](https://amp.dev/documentation/guides-and-tutorials/develop/media_iframes_3p/?format=websites#images).',
```

I would also consider "load lazily" instead of "lazy-load" since that is a domain specific technical term. It could be difficult to translate.

Reply:
I don't recall any issues with lazy-load for this https://github.com/GoogleChrome/lighthouse/blob/7d685217df0083bc69ffd2e8360ea7a75dd8dbb9/lighthouse-core/audits/byte-efficiency/offscreen-images.js#L24 so i think we good